src/tex/appendix.tex

```diff
@@ -49,6 +49,100 @@ \subsection{Benchmark corpus}
+        & & D. $S_N2$ \\
+        \midrule
+        Toxicity/Safety & 24.2 & Pindolol and propranolol are (relatively nonselective) antagonists at $\beta_1$- and $\beta_2$-adrenoceptors. However, pindolol is a partial agonist, whereas propranolol is a pure antagonist. What follows from this? \\
+        & & A. Pindolol has a greater therapeutic range than propranolol \\
+        & & B. Pindolol has a longer half-life than propranolol \\
+        & & C. Pindolol has intrinsic activity \\
+        & & D. Pindolol is more lipophilic than propranolol \\
+        & & E. Pindolol is more potent than propranolol \\
+        \midrule
+        Analytical Chemistry & 5.8 & Which of the following analytical methods is most appropriate for performing a survey analysis of a solid sample containing various metals? \\
+        & & A. X-ray fluorescence analysis \\
+        & & B. Differential pulse polarography \\
+        & & C. Flame-atomic absorption spectroscopy \\
+        & & D. Gas chromatography with flame ionization detector \\
+        & & E. Hydride generation atomic absorption spectroscopy \\
+        \midrule
+        Materials Science & 3.1 & For NMR analysis, you need to digest the MOF in a strong acid to remove the linker and leave the metal clusters intact. Why would one choose \\ce{HF} over \\ce{HCl} for this purpose? \\
+        & & A. \\ce{F-} forms a stable bonds to the metal ions \\
+        & & B. \\ce{HF} has a better water solubility than \\ce{HCl} \\
+        & & C. \\ce{HF} has a higher boiling point than \\ce{HCl} \\
+        & & D. \\ce{HF} is a weaker acid than \\ce{HCl} \\
+        \midrule
+        General Chemistry & 5.3 & Which of the following salts is an acidic salt? \\
+        & & A. \\ce{NH4Cl} \\
+        & & B. \\ce{Na2CO3} \\
+        & & C. \\ce{NaH2PO4} \\
+        & & D. \\ce{Zn(OH)Cl} \\
+        \midrule
+        Physical Chemistry & 6.3 & The Born-Oppenheimer (BO) approximation is widely used in computational chemistry, but its accuracy can vary depending on the system. Among the following options, for which system is the Born-Oppenheimer approximation likely to be least applicable? \\
+        & & A. \\ce{C60} \\
+        & & B. \\ce{CH4} \\
+        & & C. \\ce{Fe(CO)5} \\
+        & & D. \\ce{H2+} \\
+        & & E. \\ce{NaCl} \\
+        \midrule
+        Inorganic Chemistry & 3.3 & What is the oxidation number of the metal in the compound \\ce{[ZrF7]^{3-}}\\
+    \end{tabular}
+    \label{tab:chembench_corpus_topic}
+\end{table*}
+
+In addition, as shown in \Cref{tab:chembench_corpus_cognitive} the \chembench corpus can be divided considering the different mental approaches needed to solve the questions.
+
+\begin{table*}[!t]
+    \centering
+        \caption{Examples for each of the mental approaches considered in the \chembench corpus.}
+    \begin{tabular}{p{2.7cm}cp{10cm}}
+        \toprule
+        \textbf{Cognitive Type} & \textbf{Number of Questions} & \textbf{Example Question} \\
```

Comment:
Chat GPT suggested this name (`Cognitive Type` or mental approaches) to define the different types of questions. I am not very convinced about it, so has anyone a better idea or suggestion @lamalab-org/chem-bench? (Note that call it knowledge type would be strange since one of the types is knowledge)

Reply:
in other parts we called is something like "required skills"

Reply:
Yep, that sounds good, thanks :)

Reply:
Done :)

Reply:
thanks!